### PR TITLE
Mild rewrite of screen alerts to enable more functionality

### PR DESCRIPTION
**Pull request by MrPerson**
Specifically I wanted to allow #12043 to correctly implement clicking an "on fire" alert to call resist without affecting other alerts which is currently impssible. Plus this whole mess was stupid and bothering me so here we go.

### CODE REVIEW

File: code/_onclick/hud/alert.dm [hunk 1]
```diff
@@ -202,30 +209,36 @@ or shoot a gun to move around via Newton's 3rd Law of motion."
...
 
 /obj/screen/alert/emptycell
 	name = "Out of Power"
 	desc = "Unit's power cell has no charge remaining. No modules available until power cell is recharged. \
 Reharging stations are available in robotics, the dormitory's bathrooms. and the AI satelite."
+	icon_state = "emptycell"
 
 /obj/screen/alert/lowcell
 	name = "Low Charge"
 	desc = "Unit's power cell is running low. Reharging stations are available in robotics, the dormitory's bathrooms. and the AI satelite."
```
Comment (Anonus) [inline code/_onclick/hud/alert.dm]: Unrelatedly this is dongs because it doesn't support any other maps seriously don't do this

Comment (MrPerson) [inline code/_onclick/hud/alert.dm]: Fix it in another pull request. I'm not gonna fix that here.

Comment (Dorsisdwarf) [inline code/_onclick/hud/alert.dm]: what do you mean, anonus.

Pretty sure every map except the mini maps has charges in robo, sat, and bathrooms

**Comment by kingofkosmos** [comment]
:+1::+1:

**Pull request merged**